src/cloud/components/onboarding/useGetUserStatus.ts

```diff
@@ -83,29 +83,67 @@ export const getUserStatus = (table: Table): USER_PILOT_USER_STATUS[] => {
   return dataStates
 }
 
+export const queryUsage = async (
+  orgID: string,
+  queryLength?: string
+): Promise<Table> => {
+  let csvToParse = ''
+  if (getOrgsUsage) {
+    if (isFlagEnabled('newUsageAPI')) {
+      const usage = await getOrgsUsage({
+        orgID,
+        query: {
+          start: queryLength ?? '-30d',
```

Comment:
dang, that's a long default:

1. How intense is the payload that's returned from a 30d query
2. Should it just be `30d`? I ask because that's the way quartz's usage API is constructed, so just checking to be sure

Reply:
2. it shouldnt
1. We are going to test that shortly, this endpoint needs to be in PROD to be able to work

Reply:
classic

Reply:
Yeah, this makes this very difficult to implement because im essentially going blind. The plan is to work with @mavarius to flip the feature flag on for just him in PROD, then see how it works and slowly expand it out from there.